### ISSUE TITLE
highlight or event lines via shiny input

### ISSUE THREAD

**Issue by pinanunes** [rstudio/dygraphs#95]
Great stuff you got here!
This is most likely a feature request:

Would it be possible to change the highlight location or a event line based on a shiny input?
This would be great fro example for space-time data analysis so one could click an event on a map and show the date on the graph.

Thanks

**Comment by jjallaire** [comment]
Yes, if you just regenerate the entire dygraph in a renderDygraph block
when the shiny input changes you should be able to get this to work
